packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -76,13 +76,13 @@ export class SidebarComponent {
 		const itemSelector = `${ selectors.sidebar } :text-is("${ item }"):visible`;
 		await this.page.dispatchEvent( itemSelector, 'click' );
 
+		// Wait for navigation after clicking the top level item.
+		await this.page.waitForNavigation( { timeout: 30 * 1000 } );
```

Comment:
While this is what it used before, I'll note that `waitForURL()` [is preferred](https://playwright.dev/docs/api/class-page#page-wait-for-navigation) over `waitForNavigation()`. I also try to use the `waitUntil` option with a value of `domcontentloaded` to speed things up when possible.